Build Failure on refs/heads/main

@gemini-code-assist /jules debug this build error

**Log:**
```
GraffitiXR/core/nativebridge/src/main/cpp/MobileGS.cpp:212:16: note: previous definition is here
C/C++:   212 | bool MobileGS::loadSuperPoint(const std::vector<uchar>& onnxBytes) {
C/C++:       |                ^
C/C++: 1 error generated.

> Task :core:nativebridge:buildCMakeDebug[arm64-v8a] FAILED
gradle/actions: Writing build results to /home/runner/work/_temp/.gradle-actions/build-results/gradle-build-1776524812596.json

[Incubating] Problems report is available at: file:///home/runner/work/GraffitiXR/GraffitiXR/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':core:nativebridge:buildCMakeDebug[arm64-v8a]'.
> com.android.ide.common.process.ProcessException: ninja: Entering directory `/home/runner/work/GraffitiXR/GraffitiXR/core/nativebridge/.cxx/Debug/4d5m5x6h/arm64-v8a'
  [1/8] Building CXX object CMakeFiles/graffitixr.dir/SurfaceUnroller.cpp.o
  [2/8] Building CXX object CMakeFiles/graffitixr.dir/StereoProcessor.cpp.o
  [3/8] Building CXX object CMakeFiles/graffitixr.dir/SurfaceMesh.cpp.o
  [4/8] Building CXX object CMakeFiles/graffitixr.dir/MobileGS.cpp.o
  FAILED: CMakeFiles/graffitixr.dir/MobileGS.cpp.o 
  /usr/local/lib/android/sdk/ndk/28.2.13676358/toolchains/llvm/prebuilt/linux-x86_64/bin/clang++ --target=aarch64-none-linux-android29 --sysroot=/usr/local/lib/android/sdk/ndk/28.2.13676358/toolchains/llvm/prebuilt/linux-x86_64/sysroot -Dgraffitixr_EXPORTS -I/home/runner/work/GraffitiXR/GraffitiXR/core/nativebridge/src/main/cpp/include -I/home/runner/work/GraffitiXR/GraffitiXR/core/nativebridge/src/main/cpp/../../../libs/glm -isystem /home/runner/work/GraffitiXR/GraffitiXR/core/nativebridge/libs/opencv/sdk/native/jni/include -g -DANDROID -fdata-sections -ffunction-sections -funwind-tables -fstack-protector-strong -no-canonical-prefixes -D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security  -std=c++17 -fno-limit-debug-info  -fPIC -MD -MT CMakeFiles/graffitixr.dir/MobileGS.cpp.o -MF CMakeFiles/graffitixr.dir/MobileGS.cpp.o.d -o CMakeFiles/graffitixr.dir/MobileGS.cpp.o -c /home/runner/work/GraffitiXR/GraffitiXR/core/nativebridge/src/main/cpp/MobileGS.cpp
  /home/runner/work/GraffitiXR/GraffitiXR/core/nativebridge/src/main/cpp/MobileGS.cpp:217:16: error: redefinition of 'loadSuperPoint'
    217 | bool MobileGS::loadSuperPoint(const std::vector<uchar>& onnxBytes) { return mSuperPoint.load(onnxBytes); }
        |                ^
  /home/runner/work/GraffitiXR/GraffitiXR/core/nativebridge/src/main/cpp/MobileGS.cpp:212:16: note: previous definition is here
    212 | bool MobileGS::loadSuperPoint(const std::vector<uchar>& onnxBytes) {
        |                ^
  1 error generated.
  [5/8] Building CXX object CMakeFiles/graffitixr.dir/VoxelHash.cpp.o
  [6/8] Building CXX object CMakeFiles/graffitixr.dir/GraffitiJNI.cpp.o
  [7/8] Building CXX object CMakeFiles/graffitixr.dir/SuperPointDetector.cpp.o
  ninja: build stopped: subcommand failed.
  
  C++ build system [build] failed while executing:
      /usr/local/lib/android/sdk/cmake/3.22.1/bin/ninja \
        -C \
        /home/runner/work/GraffitiXR/GraffitiXR/core/nativebridge/.cxx/Debug/4d5m5x6h/arm64-v8a \
        graffitixr
    from /home/runner/work/GraffitiXR/GraffitiXR/core/nativebridge

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights from a Build Scan (powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.4.0-rc-1/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 36s
321 actionable tasks: 157 executed, 164 from cache

```

--- a/core/nativebridge/src/main/cpp/MobileGS.cpp
+++ b/core/nativebridge/src/main/cpp/MobileGS.cpp
@@ -209,9 +209,6 @@ void MobileGS::setViewportSize(int w, int h) { mScreenWidth = w; mScreenHeight =
 void MobileGS::setRelocEnabled(bool e) { mRelocEnabled = e; }
 void MobileGS::restoreWallFingerprint(const cv::Mat& d, const std::vector<cv::Point3f>& p) { mWallDescriptors = d.clone(); mWallKeypoints3D = p; }
 void MobileGS::scheduleRelocCheck(const cv::Mat& f) { mRelocColorFrame = f.clone(); mRelocRequested = true; mRelocCv.notify_one(); }
-bool MobileGS::loadSuperPoint(const std::vector<uchar>& onnxBytes) {
-    return mSuperPoint.load(onnxBytes);
-}
 void MobileGS::setArtworkFingerprint(const cv::Mat& c, const uint8_t* d, int w, int h, int s, const float* i, const float* v) {}
 MobileGS::FingerprintData MobileGS::generateFingerprint(const cv::Mat& i, const cv::Mat& m, const uint8_t* d, int w, int h, int s, const float* intr, const float* v) { return {}; }
 bool MobileGS::loadSuperPoint(const std::vector<uchar>& onnxBytes) { return mSuperPoint.load(onnxBytes); }

